the-collab-lab/tcl-45-smart-shopping-list
# About page

## Description



🚧 Work in Progress 🚧 but your opinion needed

I have created the About page, which will contain basic info about the app, instructions on how to use it, names of our mentors and our names.

The page is not completed, but I wanted you to have a look at it, check the page content and help correct phrases whenever it is needed, share thoughts on the layout and the color scheme.

To be added later:
1. About button (with a question mark instead of words) to the Home screen
2. About button to the top bar displayed in the List and Add Item pages
3. Links to our Github and LinkedIn profiles in the Creators section of About
4. Return button to go back from the About page

## Related Issue



Closes #53 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates
<img width="739" alt="Screen Shot of About page for Shop-Ade app" src="https://user-images.githubusercontent.com/63440229/187016646-a08813bb-dde1-4eea-91f7-6d1a382add83.png">

## Testing Steps / QA Criteria



Check out `NZ-about-page` and run `npm start`. To access the page add `/about` after localhost
